### ISSUE TITLE
Module parse failed: Unexpected character '�' (1:0) in @remotion/compositor-darwin-arm64/compositor

### ISSUE THREAD

**Issue by Andreaswt** [remotion-dev/remotion#3303]
After installing the NPM package @remotion/renderer, and importing the following in a server action:
`import { getVideoMetadata, VideoMetadata } from "@remotion/renderer";`

And calling getVideoMetadata:
`const videoMetadata: VideoMetadata = await getVideoMetadata(video.bucketPath);`

I receive the following error:
<img width="855" alt="Skærmbillede 2023-12-27 kl  22 24 43" src="https://github.com/remotion-dev/remotion/assets/47352649/5d2fd94d-bad1-4be5-ab8e-c0881ef14336">

I get these errors in the client console:
<img width="937" alt="Skærmbillede 2023-12-27 kl  22 25 22" src="https://github.com/remotion-dev/remotion/assets/47352649/c3b50f6c-3f97-48f8-b265-1732398ea5a9">

These are the remotion packages already installed in the project:
<img width="315" alt="Skærmbillede 2023-12-27 kl  22 23 21" src="https://github.com/remotion-dev/remotion/assets/47352649/4068860a-316b-4671-b28a-28c776c307c5">

It is a next.js v.14.0.4 project using the app router. 
Code is running on a Apple M2 Pro MacBook, with Sonoma 14.0.

**Comment by JonnyBurger** [comment]
Indeed, it does not work when you try to bundle it. I wonder if we can improve that.

**Comment by slavovthinks** [comment]
Is there any solution to that problem?

**Comment by JonnyBurger** [comment]
The solution is to use Mediabunny for getting metadata ☺️ https://www.remotion.dev/docs/mediabunny/metadata
